src/v3/src/transformer/layout/securityQuestion/securityQuestionEnroll.ts

```diff
@@ -214,7 +214,7 @@ export const transformSecurityQuestionEnroll: IdxStepTransformer = ({ transactio
 
   // update default dataSchema
   dataSchema.submit = predefinedSubmitButton.options;
-  dataSchema.fieldsToValidate = ['credentials.answer'];
+  dataSchema.fieldsToValidate = ['credentials.answer', QUESTION_KEY_INPUT_NAME];
```

Comment:
Since the predefined question dropdown has an "empty" option and since it is a required field, we have to add it to the array of fields that need client side validation.

Reply:
This is so the user has to pick something other than the "empty" option?

Reply:
Yessir exactly! Previously, if they selected the empty option, it would allow submission (with the empty value) and fail (instead of preventing it with the client-side error).